db/migrate/20220129225619_remove_column.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveColumn < ActiveRecord::Migration[6.1]
```

Comment:
`RemoveColumn` doesn't tell me much about what this migration is trying to do. Remember to be descriptive with migrations names, for example:

- `CreateUsersTable`
- `AddUserRoleToUsersTable`
- `RemoveUserRoleFromUsersTable`
- etc

Reply:
Done